packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -346,12 +346,6 @@ const Chart = props => {
     return (
         <SmartChartWithRef
             ref={charts_ref}
-            initialData={{
-                activeSymbols: active_symbols,
-            }}
-            chartData={{
```

Comment:
please add this component to qa checklist

Reply:
sure